### DISCUSSION TITLE
大佬们，这是咋回事

### DISCUSSION THREAD

**Discussion by Xingzs86** [AlistGo/alist#6837] [Q&A]
总是提示，不是内部或外部命令，也不是可运行的程序或批处理文件![image](https://github.com/user-attachments/assets/ff874178-c361-408f-9327-3700db34bef7)

**Comment by Terminal-Void** [comment]
法1:配置环境变量
法2:在下载好的文件夹中打开cmd运行命令或使用cd命令进入下载好的文件夹
法3:使用绝对路径运行alist

**Comment by Simonzhang8** [comment]
你要在cd到alist程序所在的路径再打开它，然后windows环境下可执行文件为alist.exe
